base/timing.jl

```diff
@@ -323,8 +324,9 @@ macro timev(msg, ex)
             compile_elapsedtime = cumulative_compile_time_ns_after() - compile_elapsedtime)
         )
         local diff = GC_Diff(gc_num(), stats)
-        isnothing($(esc(msg))) || print($(esc(msg)), " ->")
-        timev_print(elapsedtime, diff, compile_elapsedtime)
+        local has_msg = !isnothing($(esc(msg)))
+        has_msg && print($(esc(msg)), " : ")
```

Comment:
I think it looks better without the first space
```suggestion
        has_msg && print($(esc(msg)), ": ")
```